catalog/task/slack-webhook-notification/0.1/slack-webhook-notification.yaml

```diff
@@ -0,0 +1,68 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "release"
+  name: slack-webhook-notification
+spec:
+  description: >-
+    Sends message to slack using incoming webhook
+  params:
+    - name: message
+      description: Message to be sent
+    - name: extraDataJsonPath
+      description: path to extraData json file
+      default: "$(workspaces.data.path)/extra_data.json"
+    - name: secretName
+      description: |
+        Secret with at least one key where value is webhook URL for slack.
+        eg. oc create secret generic my-secret --from-literal team1=https://hooks.slack.com/services/XXX/XXXXXX 
+        --from-literal team2=https://hooks.slack.com/services/YYY/YYYYYY
+      default: slack-webhook-notification-secret
+  volumes:
+    - name: webhook-secret
+      secret:
+        secretName: $(params.secretName)
+        optional: true
+  steps:
+    - name: send-message
+      image: >-
+        quay.io/hacbs-release/release-utils@sha256:5733ece907aa70d6ebced36484c936731e8b27bfcf87fed226a0ecf95047a6b8
+      volumeMounts:
+        - name: webhook-secret
+          mountPath: "/etc/secrets"
+          readOnly: true
+      env:
+        - name: HOME
+          value: /tekton/home
+        - name: MESSAGE
+          value: $(params.message)
+      script: |
+        #!/usr/bin/env bash
+        set -x
+
+        # get keyName from ExtraData
+        echo "extraData: $(params.extraDataJsonPath)"
+        cat "$(params.extraDataJsonPath)"
```

Comment:
similar comment... we cat it on the next line. Is this just for debugging? I would think line 51 would cat the path and show its contents

Reply:
fixed